src/greyVest/NestedPicker.stories.js

```diff
@@ -3,12 +3,10 @@ import { storiesOf } from '@storybook/react'
 import { action } from '@storybook/addon-actions'
 import { NestedPicker } from '.'
 import decorator from './stories/decorator'
+import { options } from '../utils/stories'
 
 storiesOf('GreyVest Library|NestedPicker', module)
   .addDecorator(decorator)
   .add('story', () => (
-    <NestedPicker
-      options={['abcd', 'bcde', 'cdef'].map(x => ({ label: x, value: x }))}
-      onChange={action(`picked`)}
-    />
+    <NestedPicker options={options} onChange={x => action(`picked`)(x)} />
```

Comment:
Is there a reason the onChange got wrapped with the extra function?  It doesn't seem necessary